components/colorwheel/stories/template.js

```diff
@@ -0,0 +1,41 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { Template as ColorHandle } from "@spectrum-css/colorhandle/stories/template.js";
+
+import '../index.css';
+
+export const Template = ({
+  rootClass = "spectrum-ColorWheel",
+  customClasses = [],
+  isDisabled = false,
+  isFocused = false,
+  colorHandleStyle = {
+    '--spectrum-picked-color': 'rgb(255, 0, 0);',
+  },
+  ...globals
+}) => {
+  return html`
+    <div class=${classMap({
+      [rootClass]: true,
+      'is-disabled': isDisabled,
+      'is-focused': isFocused,
+      ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
+  })}>
+  <div class="${rootClass}-inner">
+    <div class="${rootClass}-colorarea-container"></div>
+  </div>
+  <div class="${rootClass}-border ${isDisabled ? 'is-disabled' : ""}">
+    <div class="${rootClass}-wheel ${isDisabled ? 'is-disabled' : ""}"></div>
```

Comment:
I don't know if this is any better or not but you could just do the `classMap` like you're doing for the div before this to conditionally add the `is-disabled` class

Reply:
Glad you send something, I had tried to use a `classMap` previously and couldn't get the syntax right, figured it out now!